pennylane/circuit_drawer/representation_resolver.py

```diff
@@ -29,9 +29,12 @@ class RepresentationResolver:
         show_variable_names (bool, optional): Show variable names instead of variable values.
     """
 
-    def __init__(self, charset=UnicodeCharSet, show_variable_names=False):
```

Comment:
Does this need to be removed here? That is, is it sufficient to leave this keyword argument, as long as it is never called in _construct?